src/js/background.js

```diff
@@ -748,6 +788,10 @@ function startBackgroundListeners() {
     });
   });
 
+  chrome.tabs.onActivated.addListener(function (activeInfo) {
```

Comment:
Can you put this inside the `startBackgrountdListeners` function?

Reply:
This is already inside `startBackgroundListeners` ...